### PR TITLE
Loosen test of compressed size in CompressionTest

### PR DESCRIPTION
**Pull request by pcanal**
libzstd-1.5.5-1.fc36.x86_64 increases size to 1162245

### CODE REVIEW

File: root/io/compression/MainEvent.cxx [hunk 1]
```diff
@@ -320,16 +320,16 @@ int main(int argc, char **argv)
...
 #ifdef R__HAS_DEFAULT_LZ4
       else if (comp == 6) expectedSize = 1285037;
 #else
       else if (comp == 6) expectedSize = 1208871;
 #endif
 
-      if (expectedSize > 0 &&
-          (tree->GetZipBytes() > expectedSize + 5000 ||
-           tree->GetZipBytes() < expectedSize - 5000)) {
+      if (true || expectedSize > 0 &&
```
Comment (Axel-Naumann) [inline root/io/compression/MainEvent.cxx:330]: Why this change?

**Review by Axel-Naumann — CHANGES_REQUESTED** [review]

File: root/io/compression/MainEvent.cxx (hunk 1 above)
Comment (pcanal) [reply root/io/compression/MainEvent.cxx:330]: sigh ... git fails me and left debugging code :(

**Review by Axel-Naumann — APPROVED** [review]
